src/header/mod.rs

```diff
@@ -164,8 +164,9 @@ impl Headers {
     /// # let mut headers = Headers::new();
     /// headers.set_raw("content-length", vec![b"5".to_vec()]);
     /// ```
-    pub fn set_raw<K: Into<Cow<'static, str>>>(&mut self, name: K, value: Vec<Vec<u8>>) {
-        self.data.insert(UniCase(CowStr(name.into())), Item::new_raw(value));
+    pub fn set_raw<'a, K: Into<Cow<'a, str>>>(&mut self, name: K, value: Vec<Vec<u8>>) {
```

Comment:
Fair enough... I'll revert this. Come to think of it, this change isn't even required for the `http2` code anymore (as I found a different way to do what this had been intended to).